packages/office-ui-fabric-react/src/utilities/selection/Selection.ts

```diff
@@ -86,7 +86,7 @@ export class Selection implements ISelection {
 
   public setModal(isModal: boolean): void {
     if (this._isModal !== isModal) {
-      this.setChangeEvents(true);
+      this.setChangeEvents(false);
```

Comment:
Would you mind adding a comment as to why this is necessary?

Reply:
Do you want the comment in all the other places?

Reply:
I should have looked more closely. For some reason I thought this was the only place the change should occur. Ignore this!